src/components/Icon.tsx

```diff
@@ -4,10 +4,11 @@ import * as icons from '@material-ui/icons'
 
 interface Icon {
     icon: keyof typeof icons
+    size: number
 }
 
-export const Icon: FunctionComponent<Icon> = icon => {
-    const IconName = icons['SearchSharp']
+export const Icon: FunctionComponent<Icon> = ({ icon, size }) => {
+    const IconName = icons[icon]
 
-    return <IconName className="mb-4 bg-violet-mist text-vivid-violet rounded p-2" style={{ fontSize: 48 }} />
+    return <IconName className="mb-4 bg-violet-mist text-vivid-violet rounded p-2" style={{ fontSize: size }} />
```

Comment:
@bretthayes Following up on your [comment in the issue](https://github.com/sourcegraph/about/issues/5432#issuecomment-1155413136), here is the proposed `Icon` component.

Reply:
Cool! I'd suggest using `name` as the prop instead of `icon` since we already know it's an icon from the name of the component 🤓